README.md

```diff
@@ -9,16 +9,16 @@ GreenplumPython provides a [pandas](https://pandas.pydata.org/)-like DataFrame A
 
 ## Installation
 
-To install the latest development version, do
+To install the latest development version as non root user, do
```

Comment:
Won't be this conflict with the below examples?

*as non root*, then install without `--user`. This would fail in the normal environment since normal user won't be able to write to the `/usr/local/`

Reply:
You're right, just added a new section for virtual env